Earthfile

```diff
@@ -378,20 +378,20 @@ base-image:
 
     DO +OSRELEASE --HOME_URL=https://github.com/kairos-io/kairos --BUG_REPORT_URL=https://github.com/kairos-io/kairos/issues --GITHUB_REPO=kairos-io/kairos --VARIANT=${VARIANT} --FLAVOR=${FLAVOR} --OS_ID=${OS_ID} --OS_LABEL=${OS_LABEL} --OS_NAME=${OS_NAME} --OS_REPO=${OS_REPO} --OS_VERSION=${OS_VERSION}
 
+    # Fully remove machine-id, it will be generated on first boot
+    RUN rm -rf /etc/machine-id
+
     # Enable services
     IF [ -f /sbin/openrc ]
-      # Fully remove machine-id, it will be generated
-      RUN rm -rf /etc/machine-id
       RUN mkdir -p /etc/runlevels/default && \
         ln -sf /etc/init.d/cos-setup-boot /etc/runlevels/default/cos-setup-boot  && \
         ln -sf /etc/init.d/cos-setup-network /etc/runlevels/default/cos-setup-network  && \
         ln -sf /etc/init.d/cos-setup-reconcile /etc/runlevels/default/cos-setup-reconcile && \
         ln -sf /etc/init.d/kairos-agent /etc/runlevels/default/kairos-agent
     # Otherwise we assume systemd
     ELSE
-      # Empty machine-id so we dont accidentally run systemd-firstboot ¬_¬
-      RUN rm -rf /etc/machine-id && touch /etc/machine-id && chmod 444 /etc/machine-id
-      RUN ls -liah /etc/systemd/system
+      # mask systemd-firstboot dont accidentally run block booting
+      RUN systemctl mask systemd-firstboot
```

Comment:
According to [the docs](https://www.freedesktop.org/software/systemd/man/systemd-firstboot.html), we are not only masking the creation of machine-id but also:

```
The following settings may be configured:

    - The machine ID of the system
    - The system locale, more specifically the two locale variables LANG= and LC_MESSAGES
    - The system keyboard map
    - The system time zone
    - The system hostname
    - The kernel command line used when installing kernel images
    - The root user's password and shell
```

If that ok? Did we rely on this for any of the other settings?

Reply:
we dont rely on anything there. systemd-firstboot is there for not initialized machine to configure the system on the first boot, but we are not a normal system, we already have all that configured in the rootfs :)